### PR TITLE
[4.0] Remove some deprecated code used on the front end

### PR DESCRIPTION
**Pull request by laoneo**
Removes some deprecated code on the front end.

### CODE REVIEW

File: plugins/system/languagefilter/languagefilter.php [hunk 1]
```diff
@@ -894,7 +894,7 @@ private function setLanguageCookie($languageCode)
 		// If not, set the user language in the session (that is already saved in a cookie).
 		else
 		{
-			Factory::getSession()->set('plg_system_languagefilter.language', $languageCode);
+			Factory::getApplication()->getSession()->set('plg_system_languagefilter.language', $languageCode);
```
Comment (SharkyKZ) [inline plugins/system/languagefilter/languagefilter.php]: Use `$this->app`?

File: plugins/system/languagefilter/languagefilter.php [hunk 2]
```diff
@@ -915,7 +915,7 @@ private function getLanguageCookie()
 		// Else get the user language from the session.
 		else
 		{
-			$languageCode = Factory::getSession()->get('plg_system_languagefilter.language');
+			$languageCode = Factory::getApplication()->getSession()->get('plg_system_languagefilter.language');
```
Comment (SharkyKZ) [inline plugins/system/languagefilter/languagefilter.php]: Use `$this->app`?

**Comment by laoneo** [comment]
Thanks @SharkyKZ

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**